.github/workflows/release-workflow.yml

```diff
@@ -18,14 +18,21 @@ jobs:
       with:
         version: 10.x
 
-    - name: 'Apply the new version numbers'
-      run: |
-        node ./scripts/run-yarn.js version apply --all
-
     - name: 'Generate the artifacts'
       run: |
         node ./scripts/run-yarn.js release:all
 
+    - id: check-if-deb-is-needed
+      name: 'Check whether the deb package must be rebuilt'
+      run: |
+        compgen -G './artifacts/yarn-*.js' > /dev/null
```

Comment:
Why do you need this?

Reply:
It prevents adding the deb package in a release unless the CLI changed (I'm still not sure whether we should rebuild & include ALL of them, or just the ones that changed - the later is significantly harder as we need to rebuild them with the same commit as the one used for the original release, unless we just download & re-upload them from the last release).

Reply:
For what it's worth, with ReactJS.NET (a C# project) we always rebuild everything even if nothing in that package changed. It was easier that way, and then all the packages can have exact version dependencies on all the other packages.

Reply:
Yep releasing everything everytime was an option, but I wanted to try to see how we could be more flexible (with the idea that if we design tools useful for us, they will be useful for others as well).

To this end I've developed the [`yarn version check`](https://next.yarnpkg.com/advanced/managing-releases) workflow (still being fine-tuned), which tells you which workspaces have been affected by your PR and ask you to decide whether you want to bump them or not during the next release cycle. So far it works quite well, we're just missing an interactive mode.